protos/gripper/gripper.proto

```diff
@@ -0,0 +1,61 @@
+syntax = "proto3";
+
+package mavsdk.rpc.gripper;
+
+option java_package = "io.mavsdk.gripper";
+option java_outer_classname = "GripperProto";
+
+service GripperService {
+    /*
+     * Gripper grab cargo.
+     */
+    rpc Grab(GrabRequest) returns(GrabResponse) {}
+
+    /*
+     * Gripper release cargo.
+     */
+    rpc Release(ReleaseRequest) returns(ReleaseResponse) {}
+}
+
+enum GripperAction {
+    GRIPPER_ACTION_RELEASE = 0;
+    GRIPPER_ACTION_GRAB = 1;
+}
+
+message GrabRequest {
+    uint32 instance = 1;
+}
+message GrabResponse {
+    GripperResult gripper_result = 1;
+}
+
+message ReleaseRequest {
+    uint32 instance = 1;
+}
+message ReleaseResponse {
+    GripperResult gripper_result = 1;
+}
+
+// Result type.
+message GripperResult {
+    // Possible results returned for action requests.
+    enum Result {
+        RESULT_UNKNOWN = 0; // Unknown result
+        RESULT_SUCCESS = 1; // Request was successful
+        RESULT_NO_SYSTEM = 2; // No system is connected
+        RESULT_CONNECTION_ERROR = 3; // Connection error
+        RESULT_BUSY = 4; // Vehicle is busy
+        RESULT_COMMAND_DENIED = 5; // Command refused by vehicle
+        RESULT_COMMAND_DENIED_LANDED_STATE_UNKNOWN = 6; // Command refused because landed state is unknown
+        RESULT_COMMAND_DENIED_NOT_LANDED = 7; // Command refused because vehicle not landed
```

Comment:
Do those make sense for a gripper? :thinking:

Reply:
Ah yes, sorry. I forgot to narrow these down. I copied the struct from `actions.proto` initially :see_no_evil: 

I have to check which responses are possible to a vehicle command.